### PR TITLE
LDAP setup options - CA certs, STARTTLS, user schema

### PR DESCRIPTION
**Pull request by HaraldWilhelmi**
Matching pull request to https://github.com/bihealth/sodar-server/pull/1799.

### CODE REVIEW

File: env.example [hunk 1]
```diff
@@ -63,8 +63,27 @@ SODAR_AUTH_LDAP_SERVER_URI=
...
+# local ones (e.g. 'user@DOMAIN') Must be all upper-case.
 SODAR_AUTH_LDAP_USERNAME_DOMAIN=
 SODAR_AUTH_LDAP_DOMAIN_PRINTABLE=
+SODAR_AUTH_LDAP_START_TLS=
+#
+# CA_CERT_FILE is required if TLS is used and your LDAP server's CA
+# is not a public one. It may be also used to restrict the check on your
+# LDAP server's certificate for security reasons to a specific CA.
+#
+# To us it:
```
Comment (mikkonie) [inline env.example:76]: This should probably say "use" :)

**Review by mikkonie — CHANGES_REQUESTED** [review]
Apart from one minor typo this looks good. Nice and thorough comments for the environment and config examples.

**Comment by HaraldWilhelmi** [comment]
To keep the commit history clean I just force-pushed a new commit. Please check.

**Review by mikkonie — APPROVED** [review]
Approved, thanks for the PR.

**Pull request merged**